src/Encoder/PayloadEncoder.php

```diff
@@ -53,7 +53,7 @@ private function convertApsToArray(Aps $aps): array
             $data['sound'] = $aps->getSound();
         }
 
-        if ($aps->getBadge()) {
+        if ($aps->getBadge() !== null) {
```

Comment:
Please use [Yoda condition](https://en.wikipedia.org/wiki/Yoda_conditions) for this condition.

```php
if (null !== $apps->getBadge())
```